File: Sorting Algorithms/Python Implementation/Bubble_Sort.py
class bubblesort:
    def __init__(self,x):
        self.x=x
    def bubblesort(self):
        x=self.x
        for i in range(len(x)):
            for j in range((len(x) - 1) - i):
                a = 0
                if (x[j] >= x[j + 1]):
                    a = x[j]
                    x[j] = x[j + 1]
                    x[j + 1] = a
        return x

x=[9,3,6,1,7,3,2]
result = bubblesort(x)
print(result.bubblesort())

File: Sorting Algorithms/Java Implementation/Bubble_sort.java
public class test{
	public static int[] bubblesort(int[] x) {
		int i=0;
		while(i<x.length) {
			for(int j=0;j<(x.length-1)-i;j++) {
				int a=0;
				if(x[j]>=x[j+1]) {
					a=x[j];
					x[j]=x[j+1];
					x[j+1]=a;
				}
			}
			i++;
		}
		return x;
	}
	public static void main(String args[]) {
		int[] x = {9,3,6,1,7,3,2};
		int [] result = bubblesort(x);
		System.out.println("The sorted array using bubble sort is "+Arrays.toString(result));		
	}
}
